cmd/volcam-config/main.go

```diff
@@ -0,0 +1,155 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+	"unicode"
+
+	"github.com/GeoNet/delta"
+)
+
+// will form the id and title for each camera
+var titles = map[string]string{
+	"ngauruhoe":       "Ngauruhoe",
+	"whiteisland":     "White Island",
+	"ruapehu":         "Ruapehu",
+	"taranakiegmont":  "Taranaki/Egmont",
+	"kermadecislands": "Kermadec Islands",
+	"tongariro":       "Tongariro",
+}
+
+// will be used to find volcanoes from captions
+var keywords = map[string][]string{
+	"ngauruhoe":       {"Ngauruhoe"},
+	"whiteisland":     {"White Island"},
+	"ruapehu":         {"Ruapehu"},
+	"taranakiegmont":  {"Taranaki"},
+	"kermadecislands": {"Raoul", "Kermadec Islands"},
+	"tongariro":       {"Tongariro"},
+}
+
+var letters = func(c rune) bool {
+	return !unicode.IsLetter(c)
+}
+
+func main() {
+
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "\n")
+		fmt.Fprintf(os.Stderr, "Build a volcam configuration file\n")
+		fmt.Fprintf(os.Stderr, "\n")
+		fmt.Fprintf(os.Stderr, "Usage:\n")
+		fmt.Fprintf(os.Stderr, "\n")
+		fmt.Fprintf(os.Stderr, "  %s [options]\n", os.Args[0])
+		fmt.Fprintf(os.Stderr, "\n")
+		fmt.Fprintf(os.Stderr, "General Options:\n")
+		fmt.Fprintf(os.Stderr, "\n")
+		flag.PrintDefaults()
+	}
+
+	var base string
+	flag.StringVar(&base, "base", "", "delta config files, will prefix camera and mount file names")
+
+	var output string
+	flag.StringVar(&output, "output", "", "output config file")
+
+	flag.Parse()
+
+	set, err := delta.NewBase(base)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var volcs []Volcam
+	for _, l := range set.Views() {
```

Comment:
An alternative naming suggestion:
`for _, v := range set.Views()`
`for _, m := range set.Mounts()`
`for _, c := range set.InstalledCameras()`
`for k, vals := range keywords {`
`for _, val := range vals {`
`for k, val := range targets {`

Reply:
Good suggestions, I've aimed to make the variables a little more readable. Have gone for longer form where it's a struct being returned and shorter for looping like variables.